Rare flaky nil pointer during test

```
info:customers: Building module
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x1005bd2ac]

goroutine 101 [running]:
github.com/TBD54566975/ftl/go-runtime/schema.analyzersWithDependencies(...)
        github.com/TBD54566975/ftl/go-runtime/schema/extract.go:291
github.com/TBD54566975/ftl/go-runtime/schema.Extract({0x14000881e40?, 0x1014eb9f3?})
        github.com/TBD54566975/ftl/go-runtime/schema/extract.go:108 +0x360
github.com/TBD54566975/ftl/go-runtime/compile.Build({0x101fa46d8, 0x140008020f0}, {0x1400044b9e0, 0x1e}, {0x14000881e40, 0x34}, 0x140002f5b00, {0x103371398, 0x14000338920})
        github.com/TBD54566975/ftl/go-runtime/compile/build.go:157 +0x878
github.com/TBD54566975/ftl/buildengine.buildGoModule({0x101fa46d8?, 0x140008020f0?}, {0x1400044b9e0?, 0x1e?}, _, {{{0x14000881e40, 0x34}, {0x140000ba8b0, 0x2}, {0x140000ba8b4, ...}, ...}, ...}, ...)
        github.com/TBD54566975/ftl/buildengine/build_go.go:13 +0x70
github.com/TBD54566975/ftl/buildengine.buildModule({0x101fa46d8, 0x14000670f60}, {0x1400044b9e0, 0x1e}, _, {{{0x14000881e40, 0x34}, {0x140000ba8b0, 0x2}, {0x140000ba8b4, ...}, ...}, ...}, ...)
        github.com/TBD54566975/ftl/buildengine/build.go:50 +0x57c
github.com/TBD54566975/ftl/buildengine.Build({0x101fa46d8?, 0x14000670f60?}, {0x1400044b9e0?, 0x0?}, _, {{{0x14000881e40, 0x34}, {0x140000ba8b0, 0x2}, {0x140000ba8b4, ...}, ...}, ...}, ...)
        github.com/TBD54566975/ftl/buildengine/build.go:27 +0x54
github.com/TBD54566975/ftl/buildengine.(*Engine).build(0x14000886e00, {0x101fa46d8, 0x14000670f60}, {0x140000bbf35, 0x5}, 0x1400032e900, 0x140000af380)
        github.com/TBD54566975/ftl/buildengine/engine.go:703 +0x350
github.com/TBD54566975/ftl/buildengine.(*Engine).tryBuild(0x14000886e00, {0x101fa46d8, 0x14000670f60}, 0x1400027a390, {0x140000bbf35, 0x5}, 0x1400032e900, 0x140000af380, 0x0)
        github.com/TBD54566975/ftl/buildengine/engine.go:671 +0x2b8
github.com/TBD54566975/ftl/buildengine.(*Engine).buildWithCallback.func2()
        github.com/TBD54566975/ftl/buildengine/engine.go:606 +0x3a0
golang.org/x/sync/errgroup.(*Group).Go.func1()
        golang.org/x/sync@v0.7.0/errgroup/errgroup.go:78 +0x58
created by golang.org/x/sync/errgroup.(*Group).Go in goroutine 1
        golang.org/x/sync@v0.7.0/errgroup/errgroup.go:75 +0x98
error: Recipe `build` failed with exit code 2
error: Recipe `test` failed with exit code 2
```

--- a/go-runtime/schema/extract.go
+++ b/go-runtime/schema/extract.go
@@ -41,43 +41,47 @@ import (
 // It is a list of lists, where each list is a round of tasks dependent on the prior round's execution (e.g. an analyzer
 // in Extractors[1] will only execute once all analyzers in Extractors[0] complete). Elements of the same list
 // should be considered unordered and may run in parallel.
-var Extractors = [][]*analysis.Analyzer{
-	{
-		initialize.Analyzer,
-		inspect.Analyzer,
-	},
-	{
-		metadata.Extractor,
-	},
-	{
-		// must run before typeenumvariant.Extractor; typeenum.Extractor determines all possible discriminator
-		// interfaces and typeenumvariant.Extractor determines any types that implement these
-		typeenum.Extractor,
-	},
-	{
-		configsecret.Extractor,
-		data.Extractor,
-		database.Extractor,
-		fsm.Extractor,
-		topic.Extractor,
-		typealias.Extractor,
-		typeenumvariant.Extractor,
-		valueenumvariant.Extractor,
-		verb.Extractor,
-	},
-	{
-		call.Extractor,
-		// must run after valueenumvariant.Extractor and typeenumvariant.Extractor;
-		// visits a node and aggregates its enum variants if present
-		enum.Extractor,
-		subscription.Extractor,
-	},
-	{
-		transitive.Extractor,
-	},
-	{
-		finalize.Analyzer,
-	},
+var Extractors [][]*analysis.Analyzer
+
+func init() {
+	Extractors = [][]*analysis.Analyzer{
+		{
+			initialize.Analyzer,
+			inspect.Analyzer,
+		},
+		{
+			metadata.Extractor,
+		},
+		{
+			// must run before typeenumvariant.Extractor; typeenum.Extractor determines all possible discriminator
+			// interfaces and typeenumvariant.Extractor determines any types that implement these
+			typeenum.Extractor,
+		},
+		{
+			configsecret.Extractor,
+			data.Extractor,
+			database.Extractor,
+			fsm.Extractor,
+			topic.Extractor,
+			typealias.Extractor,
+			typeenumvariant.Extractor,
+			valueenumvariant.Extractor,
+			verb.Extractor,
+		},
+		{
+			call.Extractor,
+			// must run after valueenumvariant.Extractor and typeenumvariant.Extractor;
+			// visits a node and aggregates its enum variants if present
+			enum.Extractor,
+			subscription.Extractor,
+		},
+		{
+			transitive.Extractor,
+		},
+		{
+			finalize.Analyzer,
+		},
+	}
 }
 
 // NativeNames is a map of top-level declarations to their native Go names.
@@ -297,7 +301,13 @@ func analyzersWithDependencies() []*analysis.Analyzer {
 func dependenciesBeforeIndex(idx int) []*analysis.Analyzer {
 	var deps []*analysis.Analyzer
 	for i := range idx {
-		deps = append(deps, Extractors[i]...)
+		for _, extractor := range Extractors[i] {
+			if extractor == nil {
+				panic(fmt.Sprintf("analyzer at Extractors[%d] not yet initialized", i))
+			}
+
+			deps = append(deps, extractor)
+		}
 	}
 	return deps
 }

